services/ui/serverless.yml

```diff
@@ -213,60 +203,19 @@ resources:
           Comment: OAI to prevent direct public access to the bucket
     CloudFrontDistribution:
       Type: AWS::CloudFront::Distribution
+      DeletionPolicy: Retain
```

Comment:
This allows us to keep this resource even after we've destroyed the serverless assets.